airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -17,6 +29,11 @@ export const DefaultView: React.FC = () => {
           : `/${RoutePaths.Workspaces}/${workspaces[0].workspaceId}`
       }
       replace
+      // exp-signup-selected-source-definition
+      {...(isSignupSourceSelectorExperiment && {
+        state: { sourceDefinitionId },
```

Comment:
I feel we might want to more often navigate to the new-source (or destination) page in the future with preselecting a connector, I feel it would be nice to have this option as an actual query parameter (not as a react router state), so that we could actually also craft those URLs manually and link to them. We could also merge that logic separate from this PR, since I feel this can stay inside without any link to this specific experiment.

Reply:
I think that would be nice if you feel that this can be useful in the future. I think that could be a separated issue (and as you are saying a different PR) and not part of this experiment.
We can have this here for now, and change it in the future when that change is made to have this experiment shipped asap.